src/data/faq.json

```diff
@@ -411,7 +411,7 @@
                                 "textblock": [
                                     "<b>Updated 30 March, 2022</b>",
                                     "Certificates of recovery are supported since version 2.5 of the Corona-Warn-App. Immediate vaccination status display for recovered people: For recovered people, starting with version 2.9, the complete vaccination protection is displayed in the Corona-Warn-App directly after the first vaccination. The two-week waiting period is eliminated for all vaccines except Johnson & Johnson.",
-                                    "As of version 2.20, validity dates are no longer displayed on certificates of recovery. Since the validity of certificates depends on local rules, the date of the first positive test on which the certificate is based is now displayed instead."
+                                    "As of version 2.20, validity dates are no longer displayed on certificates of recovery. Since the validity of certificates depends on local rules, the date of the first positive test on which the certificate is based is now displayed instead. However, the validity dates of your recovery certificate are not lost. Although they are no longer displayed, they are still available in the certificate. So you can still view the validity data of your recovery certificate when you export your certificate as a PDF."

```

Comment:
"when you export" should be "if you export" since you can't assume that the user will actually export.

Reply:
Alright, thank you!